app/models/serializers/work_queue/decision_review_task_serializer.rb

```diff
@@ -74,7 +74,8 @@ def self.representative_tz(object)
       issueCount: issue_count(object),
       activeOrDecidedRequestIssues:
         skip_acquiring_request_issues || request_issues(object).active_or_decided.map(&:serialize),
-      appellant_type: decision_review(object).claimant&.type
+      appellant_type: decision_review(object).claimant&.type,
+      type: decision_review(object).type
```

Comment:
Technically type is already being serialized down on line 134
```ruby
  attribute :type do |object|
    decision_review(object).is_a?(Appeal) ? "Board Grant" : decision_review(object).class.review_title
  end
```
The problem with this is that HigherLevelReview does not have a type method since isn't an STI table and so this is probably going to error for HLRs or Appeals.
```suggestion
      type: decision_review(object).try(:type)
```